docs/plugins.md

```diff
@@ -149,6 +155,33 @@ function embed(
 
 If you don't want to switch to a different parser, simply return `null` or `undefined`.
 
+### `options`
+
+`options` is an object containing the custom options your plugin supports.
+
+Example:
+
+```js
+options: {
+  openingBraceNewLine: {
+    type: "boolean",
+    category: "Global",
+    default: true,
+    description: "Move open brace for code blocks onto new line."
+  }
+}
+```
```

Comment:
Can you document each key in the option?

Reply:
Of course! Can you help my out with the `category` option? I'll do this in a new PR.